prometheus/config.go

```diff
@@ -1,9 +1,11 @@
 package prometheus
 
 type Config struct {
-	Host      string `koanf:"host"`
-	Port      int    `koanf:"port"`
-	Namespace string `koanf:"namespace"`
+	Host        string `koanf:"host"`
+	Port        int    `koanf:"port"`
+	Namespace   string `koanf:"namespace"`
+	TLSCertFile string `koanf:"tlsCertificate"`
+	TLSKeyFile  string `koanf:"tlsKey"`
```

Comment:
I tried to update this into your branch, but I didn't have permissions to do changes. A minor change for consistency with other TLS configs (kafka.tls)

```suggestion
	TLSCertFile string `koanf:"tlsCertFilepath"`
	TLSKeyFile  string `koanf:"tlsKeyFilepath"`
```